apps/backend/src/authn/oidc.strategy.ts

```diff
@@ -35,13 +40,18 @@ export class OidcStrategy extends PassportStrategy(Strategy, 'oidc') {
         scope: 'openid profile email'
       },
       async function (
+        issuer: string,
+        uiProfile: OIDCProfile, //uiProfile
+        idProfile: object,
+        context: object,
+        idToken: string,
         _accessToken: string,
         _refreshToken: string,
-        profile: OIDCProfile,
+        params: object,
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        done: any
+        done: any //shouldn't be any according to linter
```

Comment:
```suggestion
        done: any
```

The linter disable comment suffices for this